contracts/story-nft/CachableNFT.sol

```diff
@@ -12,23 +12,37 @@ import { EnumerableMap } from "@openzeppelin/contracts/utils/structs/EnumerableM
 import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 abstract contract CachableNFT is OwnableUpgradeable {
     using EnumerableMap for EnumerableMap.UintToAddressMap;
+
+    enum CacheMode {
+        Passthrough,
+        Cache,
+        Auto
+    }
+
     /// @dev Storage structure for the CacheableNFT
     /// @custom:storage-location erc7201:story-protocol-periphery.CacheableNFT
     struct CacheableNFTStorage {
         // tokenId => ipId
         EnumerableMap.UintToAddressMap cache;
-        bool cacheMode;
+        bool DEPRECATED_cacheMode;
```

Comment:
Why can't we just remove this`bool DEPRECATED_cacheMode;`

Reply:
It will break upgradeability.